client/src/screens/HomeScreen.tsx

```diff
@@ -94,10 +94,11 @@ export const HomeScreen = ({ navigation }: HomeScreenProps) => {
                     {
                         categories?.map((category: Category) => {
                             return <ClickableBox name={category.name}
-                                imageUrl={category.imageUrl}
-                                onPress={() => navigation.navigate('Category', { category })}
-                                hasTtsIcon={false}
-                                key={category._id} />
+                                                 itemId={category._id}
```

Comment:
Pretty confusing name because `itemId` refers to `Item`, but here it's actually the category's id.
I would rename the prop to `id`

Reply:
done